### PR TITLE
doc: Restructure Radio test sample

### PR DESCRIPTION
**Pull request by richabp**
Restructuring the Radio test sample documentation
to make it consistent with the structure of other samples.

Signed-off-by: Richa Pandey <richa.pandey@nordicsemi.no>

### CODE REVIEW

File: samples/wifi/radio_test/sample_description.rst [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+            * The frame duration can be calculated using the formula::
+
+                  D = ((L * 8) / R ) + P
+
+                  where::
+
+                  D : Frame duration (us)
+                  L : Frame length (bytes)
+                  R : Data rate (Mbps)
+                  P : PHY overhead duration (us) (Values: 24us - Legacy OFDM, 192us - DSSS)
```
Comment (umapraseeda) [inline samples/wifi/radio_test/sample_description.rst:174]:
```suggestion
            * The frame duration can be calculated using the formula:

              .. code-block::
      
                 D = ((L * 8) / R ) + P

               where the following parameters are used:

               * D - Frame duration (us)
               * L - Frame length (bytes)
               * R - Data rate (Mbps)
               * P - PHY overhead duration (us) (Values: 24us - Legacy OFDM, 192us - DSSS)
```

File: samples/wifi/radio_test/sample_description.rst [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+See also :ref:`cmake_options` for instructions on how to provide CMake options.
+
+Testing
+=======
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Test the sample either by configuring the Wi-Fi radio or by programming the FICR parameters by running the following commands:
```
Comment (umapraseeda) [inline samples/wifi/radio_test/sample_description.rst:81]:
```suggestion
#. Test the sample either by configuring the Wi-Fi radio or by programming the FICR parameters:
```

File: samples/wifi/radio_test/sample_description.rst [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+              chnl_primary = 1
+              tx_pkt_num = -1
+              tx_pkt_len = 1400
+              tx_power = 0
+              he_ltf = 0
+              he_gi = 0
+              tx = 0
+              rx = 0
+
+         * To run a continuous Orthogonal frequency-division mulitplexing (OFDM) TX traffic sequence with the following configuration:
```
Comment (umapraseeda) [inline samples/wifi/radio_test/sample_description.rst:119]:
```suggestion
         * To run a continuous Orthogonal frequency-division multiplexing (OFDM) TX traffic sequence with the following configuration:
```

**Review by bama-nordic — APPROVED** [review]

File: samples/wifi/radio_test/ficr.rst [hunk 4]
```diff
@@ -1,49 +1,14 @@
...
-``wifi_radio_ficr_prog`` is the Wi-Fi radio ficr_prog command and it supports the following subcommands:
+``wifi_radio_ficr_prog`` is the Wi-Fi radio FICR programming command and it supports the following subcommands:
 
 .. _wifi_radio_ficr_prog_subcmds:
 
 Wi-Fi radio FICR subcommands
-============================
+****************************
 
 .. list-table:: Wi-Fi radio ficr subcommands
```
Comment (sachinthegreen) [inline samples/wifi/radio_test/ficr.rst:13]: nit: Should ficr be FICR here?

**Review by sachinthegreen — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**